src/client_builder.rs

```diff
@@ -0,0 +1,106 @@
+use crate::{Client, Result};
+use std::net::TcpStream;
+
+#[cfg(feature = "native-tls")]
```

Comment:
Given that the feature is called `tls` (the dependency is `native-tls`), I think this should be `tls` everywhere. Or we should re-jig the `Cargo.toml` so that the feature is called `native-tls`.

Reply:
Oops. My bad. Will use `tls` everywhere.